ndk/CHANGELOG.md

```diff
@@ -6,7 +6,7 @@
 - **Breaking:** Upgrade to [`ndk-sys 0.4.0`](../ndk-sys/CHANGELOG.md#040-TODO-YET-UNRELEASED) and use new `enum` newtype wrappers. (#245)
 - native_window: Use `release`/`acquire` for `Drop` and `Clone` respectively. (#207)
 - **Breaking:** audio: Rename from `aaudio` to `audio` and drop `A` prefix. (#273)
-- Implement `HasRawWindowHandle` directly on `NativeWindow`. (#274)
+- Implement `raw_window_handle` directly on `NativeWindow`. (#319)
```

Comment:
```suggestion
- Implement `raw_window_handle` directly on `NativeWindow`. (#273, #319)
```

Reply:
I don't think that we need a first PR hash here given that it  this invalidates the old one, no?